packages/interfaces/src/protocols.ts

```diff
@@ -61,10 +61,16 @@ export type ProtocolCreateOptions = {
    * - WakuRelay to receive, route and send messages,
    * - WakuLightPush to send messages,
    * - WakuStore to retrieve messages.
-   * See [Waku v2 Topic Usage Recommendations](https://rfc.vac.dev/spec/23/) for details.
+   * See [Waku v2 Topic Usage Recommendations](https://github.com/vacp2p/rfc-index/blob/main/waku/informational/23/topics.md) for details.
    *
    */
   shardInfo?: Partial<ShardingParams>;
+  /**
+   * Content topics are used to determine pubsubTopics
+   * If not provided pubsubTopics will be determined based on shardInfo
+   * See [Waku v2 Topic Usage Recommendations](https://github.com/vacp2p/rfc-index/blob/main/waku/informational/23/topics.md) for details.
+   */
+  contentTopics?: string[];
```

Comment:
A follow up to this PR will be:
- deprecate or find better way of handling `pubsubTopics` parameter;
- refine public API of it;
- set default values to TWN fleet;

Will decouple into an issue.